### ISSUE TITLE
Develop MTurk survey instrument v4

### ISSUE THREAD

**Issue by jackblun** [huntallcott/Lotteries#24]
The goal of this issue is to develop a new survey instrument to be implemented through Amazon MTurk.

**Comment by jackblun** [comment]
Latest survey instrument (work in progress) is here: https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit#

**Comment by jackblun** [comment]
I've got a rough draft of the new MTurk survey here (also in the Dropbox TESS folder): https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit#heading=h.osp19s8uo8ig

I've added a bunch of stuff, including a new set of choices involving scratch-offs at the end, and some questions seeking to get more specifically at overconfidence. I've flagged the new sections with comments.

Comments and suggestions welcome, I'm sure some of this could be improved.

**Comment by huntallcott** [comment]
@jackblun thanks for doing this and pushing it forward!!!!!!!!!!!
I left a bunch of comments in the google doc. Ben, can you look at this next? 
In addition, several other big-pictures comments:
1. I think it's generally too long -- TESS is so expensive that we need to focus on cutting down. In addition, that length will probably reduce quality of responses.
2. Ben can you look at Section 9 and make sure it maps to exactly what you have in mind for the model? I worry that with these four tables we will get some people confused.

Thanks again.

**Comment by bblockwood** [comment]
Absolutely! Thanks @jackblun, I'll go through it this weekend.

**Comment by bblockwood** [comment]
Thanks @jackblun and @huntallcott for your work on the survey so far. I have just gone through and made a bunch of comments. I suggested a few high-level changes: 

1. Retaining @jackblun's section 8c for overconfidence, but moving it after the MPL elicitation. 
2. Consistent with @huntallcott's suggestion, implement two versions of the survey, one for a lotto and one for a scratch-off. Clearly describe that lottery type early on, with screenshots as in the current Section 9, and then use that lottery for the MPL elicitation, the overconfidence portion, and the expected value treatment.

I confirm that Section 9 maps exactly to what I have in mind for the estimation of bias in the theory section. I have now manipulated things to derive an explicit lower bound on bias as a function of WTP for a blind lottery ticket — it turns out to be quite simple, see Equation (14) [here](https://github.com/huntallcott/Lotteries/blob/c243d30e2264e692d2665f72d39e9716eb501be8/Output/Lyx/Paper.pdf). 

@jackblun and @huntallcott, want to see if you agree/disagree with the comments I made, and @jackblun can accept/implement them if so? Then we can pass to @dtaubinsky to weigh in..

**Comment by huntallcott** [comment]
@bblockwood thanks for doing this and for your continued work on the model.
1. I disagree and just left you some comments.

Let me know what you think, and then I agree we can go back to @jackblun for an overhaul, then to @dtaubinsky.

**Comment by bblockwood** [comment]
Thanks @huntallcott, the comments are great. On 1 (the issue of 8c) let's pick up here, since the gdoc sidebar is getting unwieldy. You wrote:

> here is my concern with this section:
> 
> 1. I don't know what it means to calculate, "What is the biggest prize you would expect them to win?" So I see that we could get an ordinal comparison by comparing answers across A, B, and C, but I don't see anything cardinal. Thus, it doesn't separate out overconfidence about one's own strategy vs. general misunderstanding of probabilities, which I think is what you wanted in your comment above.
> 2. More importantly, I think the "own strategy" stuff is actually much deeper than picking numbers of the sort that we're doing here. For lotto, it's when to play, what store to buy at, and how much to play when, etc. For scratchoffs, it's whether to buy the last card in the pack, the first card, what games are most profitable. So we're again not going to be able to separate overconfidence in own strategy vs. misunderstanding of probabilities.
> 
> I thus stick with my comment above that we should just ask one question about their expected winnings over a long time of playing.

I think it's easier to respond in reverse:

(2) I hear you on wanting to allow for lots of flexibility in lotto playing behavior. But as we discussed on our last call, I think it's hard to know how a respondent interprets "how much money you would expect to win back on average?", and the question admits very different responses if people report their perceived average winnings or modal winnings. I don't think we can count on respondents reading "expect" in a technical sense, especially those who aren't in the expected value treatment. (Imagine if the wording instead said "how much money would you anticipate winning back?") So I continue to think it would be useful to find a question that can get at overconfidence without this issue of interpretation. I thought @jackblun's proposed question in 8.c was a reasonable approach. It's not perfect, for the reasons you note, but I think responses to it would be easier to interpret than self-reported expected winnings. 
(1) I thought "What is the biggest prize you would expect them to win?" had a straightforward interpretation — I also suggested slightly revised wording "What is the biggest prize you would expect them to win by the end of that time? (Imagine that you would receive a small prize if you guessed correctly.)" which was, in my view, even clearer. (Isn't the answer $10? By my calc there's a 54% chance that would be the highest prize won.) If we wanted to make it finer, we could also ask for their best guess about *how many* times that highest prize was won.

**Comment by huntallcott** [comment]
@bblockwood gracias amigo.
Reactions to your excellent post:
2. I think that the broad class of overconfidence is MUCH bigger than the narrow overconfidence around being a better number picker on a specific game that the experimenter picks, so these questions in 8.C are really missing the boat. But maybe they are useful as IVs to correct for measurement error, although you can see that the RST referees didn't find that part so interesting.
1. If your objection to my question wording is that different people interpret "expect" differently, then it seems like your proposed questions have exactly the same problem AND miss the boat on overconfidence per (2). However, if we do go down this path, here's another idea building off your *how many times* idea that doesn't have the skewness problem:
   - Imagine an average person played four times a month over ten years, so 480 times in total. How many times would you expect them to win *any* prize? (Correct answer is 480 * 1/24 = 20.) 
   - Imagine *you* played four times a month over ten years, so 480 times in total. How many times would you expect to win *any* prize? 
3. Another way to get at overconfidence would be to "brute force it," e.g. by asking: "Different people like to play different games in different ways. Over the long run, how much more or less of your lottery ticket costs do you think you would win back, relative to the average lottery player in your state?" (50% less, 20% less, 10% less, 0% less, 10% more, 20% more, 50% more, whatever)

Thoughts?

**Comment by bblockwood** [comment]
@huntallcott, thank you, these are good ideas. You're right, my concern is about the interpretation of "expect." [OED](https://en.oxforddictionaries.com/definition/expect)'s first definition is "to regard as likely to happen," which I think is ok for the highest prize question, but problematic if we want expected value. (In the extreme, suppose we gave them four $1 tickets and asked "how much do you expect to win on average?" — their expected winnings are $2, but their "most likely" winnings are lower because of the skewness.)

I like your "brute force" idea. Building on it... what if we split it into two questions? I'm thinking something like "Different people like to play lottery games in different ways. Over the long run, for every dollar you spend on the lottery, how much do you expect you'll win back on average? How much do you expect that the average lottery player in your state wins back on average?" This still has the skewness/interpretation issue, but as long as they interpret "expect" the same way in each sentence, it should give a clear measure of overconfidence. What do you think?

**Comment by jackblun** [comment]
Thanks @bblockwood and @huntallcott for all the discussion. I've changed the doc up to and including Section 5, where there is a lot of agreement. Agreed that it might make sense to have two completely distinct surveys, one structured around scratch-offs and one around lottos.

On the issue of expectations, one important bit of feedback from the first few mturk surveys was that people saw the expected value calculation as a mathematical exercise, and didn't always get the relation to lottery tickets. We even had cases where people argued with our interpretation and said 'no, I got it correct, the most likely outcome is that I win nothing'. I think this is something really important that we need to get as much information on from the pilots. I agree that the suggestions above are better than my rather narrow form of overconfidence.

From section 6 onwards, are we happy with the structure being?:
6. EV treatment (integrating aspects of previous section 7 on lottery choices)
7. Overconfidence (formerly 8)
8. MPL choices / anticipatory utility (formerly 9)

**Comment by huntallcott** [comment]
@jackblun @bblockwood:
1. Regarding the Overconfidence section: 
* I vote we ask the following two questions for sure, per @bblockwood's suggestion:
   - Different people like to play lottery games in different ways. Over the long run, for every dollar you spend on the lottery, how much do you expect you'll win back on average? 
  - How much do you expect that the average lottery player in your state wins back on average?
* I'm not against asking the following two questions in the context of the lotto or scratch-off game
   - Imagine an average person played four times a month over ten years, so 480 times in total. How many times would you expect them to win any prize? (Correct answer for the lotto is 480 * 1/24 = 20.)
   - Imagine you played four times a month over ten years, so 480 times in total. How many times would you expect to win any prize?
* I would vote we drop the rest of the overconfidence section. I don't think it picks up the right kind of overconfidence and even within the kind of overconfidence this might pick up, I don't think it's that material to distinguish between overconfidence about probability vs. expected winnings. In the end, people can read what the jackpot is, so overconfidence about expected winnings has to come from overconfidence about probability.

2. On the order: 
* I think we need the EV treatment to come directly before the MPLs so that the impacts are maximized. 
* I think the Overconfidence elicitation might kind of seem like an EV calculation, so I'd suggest we do it last.
* I don't think we're getting anything out of Section 7 and would drop

3. On the EV treatment itself
* I fear it's too distant. I know we're worried about demand effects, but I think it's crucial to make this fun, interesting, interactive, possibly video-based, and relevant to the MPLs.
* I think we want to target this around the reasons why some people say lotteries are a "stupidity tax." I think there are three reasons why people say that:
  - EV is low.
  - A lot of that EV is allocated to low marginal utility states. Put differently, in the typical higher-marginal utility state, the most likely outcome is that you get nothing.
   - Overconfidence: people think they have a good strategy for the lottery but in fact do not.
* Here is another proposal, just to brainstorm
   - To make the EV point: first give a quick example of calculating EV, then have a graphic that calculates the EV of the MM / scratchoff ticket they are considering. Then have people drag and drop this ticket vs. other investments (savings account for 1 year, S&P 500 for 1 average year, Treasuries for 1 average year, other pure gambles such as sports betting) to put them in order of expected returns. This makes the point that the lottery is very low-return.
   - To make the probability of winning point: do another drag and drop to place the probability of winning different prize levels in order against other things that we can document from data, e.g. shark attacks, struck by lightening, prediction market stuff (Trump winning presidency, Red Sox world series, etc), death by car accident, death by drowning in bathtub, etc. (Aside: for the average American, dying in a car accident this year is more likely than winning $200 in a MM ticket!)
   - I don't know how to disabuse people of their dumb strategies but we could think more. 

The hope is that these ideas can prove fun, interactive, and relevant. And I think because we're not TELLING people that the lottery is dumb per se, the demand effect issue is less severe than other ideas we've had.

4. Logistical issue: Remind me -- aren't we still planning to buy the tickets and distribute them to people? How are we implementing Table 3 in a credible way for TESS? Are we recording your friend's address? Incidentally, the blind lottery is more credible when we do this for lower-value scratchoffs compared to lotto.

**Comment by bblockwood** [comment]
Re @huntallcott's good logistical question:

4. I think we don't want the blind ticket recipient to be a friend of theirs. How about this? "These payments and blind lottery tickets will really be awarded. [Details:] they will be administered by Wharton School research staff members. Recipient names have been selected from the Philadelphia area, and will be randomly paired with any blind lottery tickets awarded. Staff members will then scratch the tickets under the supervision of a witness from the University of Pennsylvania Police Department, and if one should be a winner, those winnings will belong to the randomly selected recipient, who will then be notified. All participants have signed legally binding contracts aggreeing to this process. The ticket is also blind to you, meaning that you will not be notified of the outcome."

Lots of possible variations on this, but does the basic design seem credible?

**Comment by huntallcott** [comment]
@bblockwood I love it.

**Comment by dtaubinsky** [comment]
@bblockwood this is great!

**Comment by jackblun** [comment]
@huntallcott @bblockwood @dtaubinsky I've updated the survey doc here in light of our discussion: https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit#

I think most sections look good and are definitely an improvement. I'm still not entirely happy with Sections 6 and 7, which may need a bit more work. One thing you guys will have insight on is how good my chosen low-probability events are reasonable. For sports betting, I thought baseball might be the best since statistics seem to play a large part in discussion around the sport.

**Comment by huntallcott** [comment]
@jackblun thanks so much for doing this. 
Sorry for the delay. DT, BBL, and I have been working on soda taxes, but we are coming back with a vengeance now and excited to get this out ASAP.
6. I like it, and it seems clear. I think we could see some directional impact, but I don’t think it’s at all fair to think of it “full debiasing.” Thus, it would generate a very loose bound on bias. So am interested to hear how Dmitry feels about this.
7. This is a great start and I like where this is going. Can you do something differently, in the following sense:

Here are some events that might or might not happen in the future. Please put these events in order of probability, from most likely to least likely.

[In Qualtrics: use Rank order question type; randomize the order of all choices. Also note my wording below is trying to be precise but approachable.]

A given ticket winning the MegaMillions $4 prize 
A given ticket winning the MegaMillions $500 prize
A given ticket winning the MegaMillions Jackpot prize
The Baltimore Orioles winning the 2019 world series (according to BleacherReport.com)
The average American dying in a car crash
The average American being struck by lighting [aside: holy crap. this is pretty likely ... you've got me worried]
[Something with probability around 1/50, that sounds a bit unlikely. E.g.: “Cory Booker being elected U.S. president in 2020 (according to prediction markets)” although ideally it would be a-political and sound even more unlikely.]

Then have a graphic with the options in the correct order, annotated with your sources. Just give the sources/facts; don’t say stuff like “These are long odds, but far greater than those of winning the Mega Millions jackpot.”


* I think we also want the investment returns one we discussed on the phone. Here's a draft of that.

Here are some things you could buy that could earn you some money in the future. Please put these things in order of financial returns per dollar of initial investment, from highest to lowest.

[Again: Rank order question type; randomize the order of all choices.]

An investment in the S&P 500 stock index (based on returns from 1990 to today).
A certificate of deposit at a standard credit union.
Playing poker with your friends (if you are all equally good at poker)
Betting at a racetrack (if you are an average bettor.)
Gambling at a casino (if you are an average gambler)
Buying a new Ford F-150, using it normally for three years, and then re-selling it. [Use KellyBlueBook.com or something. Make sure whatever you choose is better than the lottery.]
Playing the average state lottery in the US. [We have some aggregate stats here, right?]
[Can we put something worse than a lottery here? If nothing else comes to mind, then what about: “Buying a new Ford F-150, using it normally for ten years, and then re-selling it”]

Then similarly have a graphic with the options in the correct order, annotated with your sources.


9. Overconfidence. This question is both a treatment (as I worried on the phone last week) AND (I now realize) could be affected by the EV treatment. Jack, can you remember to test for this? If we see treatment effects, then we should talk, and perhaps consider moving to be the final questions in Section 4.

**Comment by huntallcott** [comment]
Also @jackblun , two other things:
A. I know this is just sematics, but conceptually let's not call section 7 the "heavy-handed" treatment. Let's think of it as "even-handed but direct."
B. Can you find something other than being struck by lighting? It's too ... heavy-handed. Would you be willing to look at the death statistics summary somewhere online, and maybe get the car deaths from there, plus maybe my 1/50 one could be "The average american dying of heart disease / cancer", plus perhaps one other really rare cause of death (e.g. I wonder if dying of plague or Ebola in the US in the past five years is slightly more likely than one MM ticket winning the jackpot)?

Thanks again.

**Comment by jackblun** [comment]
@huntallcott thanks. I have two big presentations early next week so won't be able to get a new iteration of the survey out before then but will integrate suggestions next week.

**Comment by anjoung** [comment]
@bblockwood Asked me to implement these changes, and I made my first go! Just wanted to flag a few things:
- Changed "struck by lightening" to "caught the plague", but the likelihood of catching the plague is much lower and want to make sure that isn't an issue
- For the 1/50 event, I chose the likelihood that an American household has exactly six members
- I wasn't sure how to answer the question about the average return for the racetracks and couldn't find any sources
- Changed the question on returns from casino games in general to specific questions about Blackjack and Roulette
- We don't have aggregate stats on state lotteries, and was wondering if I should insert something about winning Powerball, MegaMillions, or a random state lottery (since it's easy to report expected payoff for an individual game)
- For my worse than a lottery investment, I chose betting on a perfect March Madness bracket which for a naive fan would occur with a probability of 1 in 9.2 quintillion.

Let me know if anything seems off. [Link to draft](https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit?usp=sharing).

**Comment by bblockwood** [comment]
Thanks @anjoung, this is a great start. A couple thoughts:

* I'd love to use a risk for 1/50 that is something that feels more haphazard than the number of members of a household, since people don't think of that as something that happens randomly. What about something like: "the probability that a randomly pulled over car happens to be a Honda Civic" (or whatever gives the right odds)?
* Regarding racetracks, Etan Green at Wharton has a new article on the favorite/longshot bias that I believe mentions the take rate at typical horse racing tracks. 
* For lottery returns, maybe we could ask about the return to a "typical lottery ticket, like Mega Millions or Powerball". (Do the expected returns differ *that* much across different games? Do you have a sense of that from the odds and prize data sets you are working on yet?)

**Comment by huntallcott** [comment]
@jackblun sorry to bother but I have quick questions on the spending questions.
1. Can you point us to the exact text of the survey where these came from? Or did we make up these questions? In general for surveys let's include a link to the relevant questionnaire if we are taking the wording from another paper.
2. You wrote that Keno is not a lotto game, but I think it technically is. (Incidentally I think that "draw games" is more common language, based on my perusal of some websites. So I think we should put Keno in the draw game category unless you disagree. So what is there in the "other" category at this point? Is it just VLT? In that case should we just say "VLT operated by your state lottery agency"?
Thanks and sorry to bother.

**Comment by huntallcott** [comment]
@anjoung thanks for doing this.
I left a bunch of comments. Take a look and address? Then let's do one more iteration before you put it into Qualtrics.
Many thanks!

@bblockwood, I left some comments in Section 8 and 9 for you. Can you take a look at those?

**Comment by anjoung** [comment]
Updated the draft. Need some clarification on a few things @huntallcott asked. 

Also, for the investment that pays out worse than the average state lottery, I picked investing in bitcoin, which in January 2018 was around 10,000 and in January 2019 was around 3,000.

**Comment by huntallcott** [comment]
Thanks @anjoung. I think I answered your questions.
So there are remaining questions for @bblockwood , @dtaubinsky, and a quick one for @jackblun. 
After that gets sorted out and you revise the instrument accordingly, I'll take a final look!

**Comment by anjoung** [comment]
@huntallcott I responded to one of your answers: "Sorry just to be clear there are other ways of playing that yield a different house advantage! They are just worse outcomes (e.g., if I play five-numbers, I get something like a 7% loss overtime)." 

Just want to make sure that was clear. Otherwise, yes everything is sorted!

**Comment by huntallcott** [comment]
I see. Just say 5-7 percent expected loss. Also if that kind of range doesn't change the ordering with the other "investments," you could also decide not to limit to "American" roulette.

**Comment by anjoung** [comment]
Got it

**Comment by jackblun** [comment]
@huntallcott sorry for the delay. On your questions above:
1. Can you point us to the exact text of the survey where these came from? Or did we make up these questions? In general for surveys let's include a link to the relevant questionnaire if we are taking the wording from another paper.

The motivation behind the choice of expenditure questions is Wood and Williams (2008) - "How Much Money Do You Spend on Gambling? The Comparative Validity of Question Wordings Used to Assess Gambling Expenditure." It is in the literature folder. They summarize in the conclusion that the monthly expenditure question comes out most reliable, though they note in general that retrospective reports of gambling expenditure tend to be unreliable.

2. You wrote that Keno is not a lotto game, but I think it technically is. (Incidentally I think that "draw games" is more common language, based on my perusal of some websites. So I think we should put Keno in the draw game category unless you disagree. So what is there in the "other" category at this point? Is it just VLT? In that case should we just say "VLT operated by your state lottery agency"?

Draw games may be a better term. I guess Keno could be considered a very fast-paced draw game and could be pooled with more conventional 'lotto' style games. In the other category, there is VLT, also more unusual games like raffles and sports sweepstakes. It might make sense to separate out VLT at least, as that is very distinct and would be clear for respondents.

**Comment by huntallcott** [comment]
@jackblun thanks for your response above.

@anjoung @bblockwood @dtaubinsky,
This survey is ready to go. @anjoung can you code it up into Qualtrics as time allows?

**Comment by huntallcott** [comment]
@rajbhargava re-assigning the Qualtrics programming to you!

**Comment by rajbhargava** [comment]
I've coded up the survey into Qualtrics [here](https://berkeley.ca1.qualtrics.com/jfe/preview/SV_1LYg4YAUri8GTxX?Q_SurveyVersionID=current&Q_CHL=preview). 

Few notes:

- For the question where we ask how much you spend on instant lotteries, draw lotteries, then total, I wasn't sure if there was a way for qualtrics to sum and compare answers. Instead, there is a page where we show all participants their answers and ask them to make sure this looks ok.
- It wasn't clear to me how assignment to control vs the various treatments is supposed to happen (eg random assignment with equal numbers to each group?) so the link above has all nine sections. I can code up the assignment scheme once I have more clarity! 
- Attempting to take the survey as a participant took me ~15 minutes.

**Comment by bblockwood** [comment]
Thanks @rajbhargava!

@anjoung, given your experience coding up Qualtrics, can you take a look at this when you have a moment, and confer with Raj about how to implement these features?

**Comment by anjoung** [comment]
Will do! @rajbhargava you might've already done this but please share this survey with my UPenn email (anjoung@upenn.edu). Just ping me once you've done this, so that I confirm that it's been shared. Qualtrics is very finicky with sharing sometimes.

**Comment by bblockwood** [comment]
Thanks @anjoung and @rajbhargava. You can make me the primary email contact for now, as we'll probably be running the MTurk version through my account with the Wharton Behavioral Lab.

**Comment by rajbhargava** [comment]
@anjoung: thanks for all the helpful feedback and for the first few notes!

Implemented the following edits:

- I changed the primary email contact to ben.lockwood@wharton.upenn.edu.
- I dropped timing blocks for all questions that are not subjective or open-ended.
- Moved the condition to deny mobile phones to survey flow.
- Changed the end of survey block to something more descriptive.
- Double-checked the hyperlinks and they are all functional.
- I left the "go back" button and the relevant js on since we (conditionally) ask them to go back and fix their answers on the question about monthly expenditures.
- Evenly randomized the treatment conditions.

**Comment by anjoung** [comment]
@rajbhargava Thanks for these changes! I think we still want to remove the "go back" button and js. My impression is that allowing this feature is useful for the expenditure questions (since it exactly matches the text), but allowing participants to return to earlier questions as they progress in the survey overall is a bad thing.

Currently, I have things coded so that if you provide an inconsistent answer in the expenditure section, I re-ask you all those questions once. My suspicion is that we wouldn't want to include people who fail this question twice (or more) anyway, so this acts as a secondary attention check.

@bblockwood What are your thoughts?

**Comment by bblockwood** [comment]
Thanks guys. I agree with @anjoung -- let's conditionally re-ask the expenditure questions a second time, and record both responses, so they're not actually overwriting any data.

**Comment by rajbhargava** [comment]
great, removed the "go back" button and all associated js.

**Comment by huntallcott** [comment]
Thanks @rajbhargava @anjoung 
1. Can you provide separate links to the different versions of the survey? I wanted to take a look at the treatments. OR a link to a version of the survey that has all treatments turned on?
2. Minor: the title says “preview - credit card usage” survey. Should this be changed?
3. I re-wrote the phrasing of “preferences over lottery play” questions. With -5 (disagree) to +5 (agree) it was not clear what the “normative consumer’s” score would be for “I should play the lottery less in the future than I do right now.” Making 0 mean “not at all” means that we can make 0 the normative consumer’s answer, in parallel to what we do with soda taxes.
Raj, unless Ben or Dmitry objects, can you make this change as I tracked it in the google doc?
4. I disagree on the validity checks. E.g. I got, “Earlier, you told us you spend $0 on instant games and $111111111 on draw games in an average month, so we imagine that you spend at least $111111111 in total. But you said that you only spend $2 in total. Something doesn't seem right. Please go back and check your answers.” So it's asking me to go back, but there’s no back button. I think for the validity checks there should be a back button and my previous answers are pre-populated for me to change.
5. Obviously, the April TK Mega Millions date should be populated

Thanks again.

**Comment by anjoung** [comment]
@rajbhargava For (4), just want to highlight that Qualtrics has a "Version History" under "Tools" so you don't have to manually re-implement everything.

**Comment by rajbhargava** [comment]
@anjoung thanks! @huntallcott:

1. [Here](https://berkeley.ca1.qualtrics.com/jfe/preview/SV_6lM9oaT8A6sxAlD?Q_SurveyVersionID=current&Q_CHL=preview) is the survey with all treatments.
2. Not sure where you are seeing that, could you clarify?
3. , 4. , 5. Done.

**Comment by huntallcott** [comment]
1. Thanks.
2. See this picture. Not a big deal but if users will see it then we should fix.
![image](https://user-images.githubusercontent.com/23505296/56770344-b795ca00-6781-11e9-9b9d-c8bd476ccf7f.png)
3. Thanks. Please accept those changes in the google doc so that it's clean.

6. Also I made other changes in the google doc in section 4 and after. Can you implement in qualtrics and accept the changes in the google doc?
7. Beginning of Section 5: I thought we were randomizing whether we say more vs. less instead of saying "more or less." @anjoung did I miss something? If not then can @rajbhargava implement that randomization?

Many thanks!

**Comment by anjoung** [comment]
@huntallcott Sorry! Thought the text was literally "[more/less]", which seemed weird so I suggested "more or less".

**Comment by rajbhargava** [comment]
@huntallcott: Fixed the name, randomized whether we say more/less, and accepted and implemented all changes you made in the google doc. Survey with all treatments is [here](https://berkeley.ca1.qualtrics.com/jfe/preview/SV_6PvTiu7zln3BChD?Q_SurveyVersionID=current&Q_CHL=preview).

**Comment by huntallcott** [comment]
OK so this has addressed all of my comments.
@bblockwood anything else? Or can we get this out and start collecting data?




Separately: @rajbhargava can you open up a separate GitHub task to create an "overview of results doc"? I think we want:
* Descriptive statistics on the sample
* Descriptive information on how people responded to each question, e.g. a table or graph of responses as you see fit.
* Treatment effects of all manipulations
* Anything else you think would be interesting.

* I think we want the analysis done in stata and imported into a lyx file, then compiled into a pdf and linked from that GitHub thread.
* This is something you could start this morning (or whenever you have time) using the randomly generated data that Qualtrics could output for you. Then you'd just update the tables/figures later with the actual data.

Many thanks in advance!

**Comment by bblockwood** [comment]
@rajbhargava thanks for your work putting this together — the survey looks quite good. I have two minor corrections to content:

1. The screenshots in the Mega Millions multiple price list section look rather fuzzy, but the graphics are very clear on [this page](https://www.megamillions.com/how-to-play). Could you see if it's possible to improve the resolution in the survey, either by replacing them or changing some setting? 
2. The instruction text at the top of that section lacks a space in the phrase "TheMega Millions..."

Then the next step is to add a few "end of survey" questions so that this works with Qualtrics. (Basically, we provide the workers with a completion code, which they enter into a field at the end of MTurk task in order to prove they've done the survey, and we also ask for their WorkerID so that we can pay them if they somehow mess up the completion code, which happens a few percent of the time.) To that end, can you please share the survey with me, and I'll add those question templates to the end? (Please share with both benlo@upenn.edu, and ben.lockwood@wharton.upenn.edu, as Qualtrics sharing is finicky and I don't remember which address is the correct one to use.) 

Many thanks for your work putting this together!

**Comment by rajbhargava** [comment]
@bblockwood made the requested changes and shared with both emails.

**Comment by bblockwood** [comment]
@rajbhargava thanks! What is your email on Qualtrics (so I can send you a survey template with the MTurk questions)?

**Comment by rajbhargava** [comment]
rajbhargava@berkeley.edu

**Comment by bblockwood** [comment]
Thanks @rajbhargava! Survey instrument is all set for now. I'm closing this issue and picking up at #25 to launch the survey itself.

**Comment by dtaubinsky** [comment]
@rajbhargava Could you share the survey with dmitry.taubinsky@berkeley.edu?

**Comment by rajbhargava** [comment]
@dtaubinsky shared!

**Comment by rajbhargava** [comment]
@bblockwood @dtaubinsky @huntallcott:

Suggested edits to the survey from the call this morning:

- Overconfidence
   * Ask about state level payouts for EV
   * Also ask about EV for $100 for themselves, vs. others
- MPL
   * Bundle anticipatory utility and joy of scratching: choice between sending another survey respondent fixed amount of money or scratch-off ticket
- Financial literacy
   * Any substitutes for financial literacy questions, given people are mostly getting them right?

One other thing I was thinking about: should the questions account for taxes at all? E.g. someone could understand pre-tax lottery winnings in expectation completely correctly but not understand how much larger tax rates on those winnings are going to be than the ones they currently face. (similarly, given evidence on small lottery winnings [not improving bankruptcy rates](https://www.mitpressjournals.org/doi/abs/10.1162/REST_a_00114), is overconfidence on outcomes conditional on winning the lottery important?)

**Comment by huntallcott** [comment]
Raj-
Thanks for this, and great idea on taxes. I think we are trying to get at
some of this through the tax rate data we are gathering on another issue
thread.
On the financial literacy questions: I talked with Dmitry and we were
wondering if you could look at the lusardi and Mitchell papers to see if
you can find any questions that might be one step harder? Maybe open up
another issue for that if you find anything that is worth debating.
Many thanks for your work and also your thinking on this stuff.
-- 

Hunt Allcott

https://sites.google.com/site/allcott/home

**Comment by rajbhargava** [comment]
Here are questions from Lusardi and Mitchell 2017 (https://doi.org/10.1142/S2010139217500082); all are from the Health and Retirement Study or the 2005 Dutch DNB Household Survey. We used which are a subset of the financial literacy questions, the "Big Three," in the last version of our pilot survey.

Basic Financial Literacy Questions
1. **Numeracy** (used in survey)
Suppose you had $100 in a savings account and the interest rate was 2% per
year. After 5 years, how much do you think you would have in the account if
you left the money to grow? (i) _More than $102_; (ii) Exactly $102; (iii) Less
than $102; (iv) Do not know (DK); (v) Refuse.
2. **Compound Interest**
Suppose you had $100 in a savings account and the interest rate is 20% per
year and you never withdraw money or interest payments. After 5 years, how
much would you have on this account in total? (i) _More than $200_;
(ii) Exactly $200; (iii) Less than $200; (iv) DK; (v) Refuse.
3. **Inflation** (used in survey)
Imagine that the interest rate on your savings account was 1% per year and
inflation was 2% per year. After 1 year, how much would you be able to buy
with the money in this account? (i) More than today; (ii) Exactly the same;
(iii) _Less than today_; (iv) DK; (v) Refuse.
4. **Time Value of Money**
Assume a friend inherits $10,000 today and his sibling inherits $10,000 3
years from now. Who is richer because of the inheritance? (i) _My friend_;
(ii) His sibling; (iii) They are equally rich; (iv) DK; (v) Refuse.
5. **Inflation/Money Illusion**
Suppose that in the year 2010, your income has doubled and prices of all
goods have doubled too. In 2010, how much will you be able to buy with your
income? (i) More than today; (ii) _The same_; (iii) Less than today; (iv) DK;
(v) Refuse.

Sophisticated Financial Literacy Questions
1. **Stock Market Functioning**
Which of the following statements describes the main function of the stock
market? (i) The stock market helps to predict stock earnings; (ii) The stock
market results in an increase in the price of stocks; (iii) _The stock market
brings people who want to buy stocks together with those who want
to sell stocks_; (iv) None of the above; (v) DK; (vi) Refuse.
2. **Knowledge of Mutual Funds**
Which of the following statements is correct? (i) Once one invests in a mutual
fund, one cannot withdraw the money in the ¯rst year; (ii) _Mutual funds
can invest in several assets, for example invest in both stocks and
bonds_; (iii) Mutual funds pay a guaranteed rate of return which depends on
their past performance; (iv) None of the above; (v) DK; (vi) Refuse.
3. **Interest Rate/Bond Prices Link**
If the interest rate falls, what should happen to bond prices? (i) _Rise_;
(ii) Fall; (iii) Stay the same; (iv) None of the above; (v) DK; (vi) Refuse.
4. **Safer: Company Stock or Mutual Fund** (used in survey)
True or false? Buying a company stock usually provides a safer return than a
stock mutual fund. (i) True; (ii) _False_; (iii) DK; (iv) Refuse.
5. **Riskier: Stocks or Bonds**
True or false? Stocks are normally riskier than bonds. (i) _True_; (ii) False; (iii)
DK; (iv) Refuse.
6. **Long Period Returns**
Considering a long time period (for example 10 or 20 years), which asset
normally gives the highest return? (i) Savings accounts; (ii) Bonds;
(iii) _Stocks_; (iv) DK; (v) Refuse.
7. **Highest Fluctuation/Volatility**
Normally, which asset displays the highest fluctuations over time? (i) Savings
accounts; (ii) Bonds; (iii) _Stocks_; (iv) DK; (v) Refuse.
8. **Risk Diversification**
When an investor spreads his money among different assets, does the risk of losing
money: (i) Increase; (ii) _Decrease_; (iii) Stay the same; (iv) DK; (v) Refuse.

**Comment by bblockwood** [comment]
@rajbhargava thanks for looking into this! I would propose the following next steps:

1. Duplicate the existing Qualtrics survey for the purposes of running a second pilot (but be careful not to over-write the previous one, as we want to retain our data from it). Please share it with all of us.
2. Implement the "Overconfidence" and "MPL" revisions discussed [above](https://github.com/huntallcott/Lotteries/issues/24#issuecomment-491922828). 
3. Let's add questions 7 and 8 from the "Sophisticated Financial Literacy" questions [just above](https://github.com/huntallcott/Lotteries/issues/24#issuecomment-492468932).

Thanks for your work pushing this forward!

**Comment by rajbhargava** [comment]
@bblockwood made the requested changes and shared with you and @dtaubinsky.

**Comment by bblockwood** [comment]
Hi @rajbhargava, thanks for duplicating the survey, and sorry for my delay getting back to you on this. I was just going to field it again, but I think I'm not seeing the edits for some reason. It is the project you shared with me, called "Lottery Survey", correct? I don't see any new questions in the "Financial Literacy" or "Overconfidence" sections. Should I be seeing the edits we discussed above?

Thanks again for your help with this.

**Comment by rajbhargava** [comment]
@bblockwood you're looking at the old survey, the duplicated one with edits is called "Lottery Pilot #2".

**Comment by bblockwood** [comment]
@rajbhargava Got it — thanks. (Had to add it manually.) This is looking good. On the overconfidence section, I think we'd still like to make a couple adjustments. Let's use the following structure for the own and average lottery players' spending questions in the overconfidence section:
<img width="647" alt="image" src="https://user-images.githubusercontent.com/956485/58585290-4a64d100-8226-11e9-89f9-6d9e4bda260e.png">

I started to make these changes, but I didn't want to mess up the javascript for the previous button, so would you mind implementing this and also adjusting the google docs version of the survey so both are consistent?

Also, did you make any changes to the MPL section? (Ok if not — the edits we discussed there were less concrete — I just want to make sure I saw any changes that were made.) Thanks again for this!

**Comment by rajbhargava** [comment]
@bblockwood made the change for overconfidence in the survey and on the gdoc; for the MPL section, I added the question titled "MPL_2".

**Comment by bblockwood** [comment]
@rajbhargava great — thanks! 

On the MPLs: I think the concern was not so much about whether the recipient is a random person vs. a survey respondent, but rather that decisions about "blind lottery tickets" feel complicated and foreign, and people might not be paying attention to the distinctions between the tables. @dtaubinsky (I believe) had proposed instead just giving people random prizes ourselves, rather than trying to implement a blind lottery. I think we can potentially do something even more simply, letting people choose between a scratch off (with anticipatory utility) and a prize drawing in which they're only notified if they win (and with a delay). I have added a sample version of the question I have in mind in the [survey gdoc](https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit#), could you take a look, then implement it for this next round, in place of our other more confusing MPL section? Then I think we will be ready to go for launching another round!

**Comment by rajbhargava** [comment]
@bblockwood: got it. Implemented the changes!

**Comment by bblockwood** [comment]
Awesome! @rajbhargava you are a machine. 

I had two more thoughts about minor improvements we could make, in the interest of boosting the effectiveness of the treatments.

1. In the sections "Expected value treatment" and "Even-handed and direct treatment", I think we should edit the "answer" pages to give more dynamic feedback on whether the subjects answers were correct. I have made some edits to the gdoc to suggest how this could look. 
2. In the section "Expected value treatment", let's add text at the beginning stating they'll get an extra $1 bonus if they get all three questions correct. (I can carry these out easily after the survey runs.)

Could you take a shot at implementing these, and let us know if you run into any issues? Implementing 1 will require some conditioning — @anjoung is a pro at this if you have any questions about it. Thank you for your efforts on this!

**Comment by rajbhargava** [comment]
@bblockwood: implemented changes. For the even-handed/direct treatment, I couldn't figure out how to make a table and so did a far more low-tech version of what you had in the gdoc!

**Comment by bblockwood** [comment]
@rajbhargava thank you for doing this! Your implementation of the even-handed/direct treatment "answers" page looks just fine. I did just preview the survey a few times and ran into some bugs — in the EV treatment, on EV_2 I didn't see an answer page, and for a later EV question I was told I had incorrectly answered "XX". In the the "even-handed direct" treatment, I think Q48 reported back a different set of rankings than I had chosen (though this was hard for me to verify). In sum, would you mind test-taking the survey several times to look for these kinds of bugs, to weed them out before we launch the survey? Thank you!

**Comment by rajbhargava** [comment]
@bblockwood apologies for the issues. The EV questions are fixed; I've double- and triple-checked the even-handed and direct treatment and it has been reporting the correct rankings.

**Comment by bblockwood** [comment]
@rajbhargava — awesome! Thanks for your diligence on this. I will launch a mini-pilot on TurkPrime now, with just 20 respondents, so we can check that everything works. Provided so, we'll launch with another 80 or so later today. I'll follow up with you once the initial batch of 20 is completed.

**Comment by rajbhargava** [comment]
@bblockwood: created and shared pilot 3 with you. It contains the following edits:

- Fixed the overconfidence section to be $100 for each question
- Added state level EV to overconfidence section
- Randomized MPL order
- Added 50/50 $2 lottery

We also discussed showing probabilities for the MPL, but I wasn't sure how/if we wanted to implement that.

**Comment by bblockwood** [comment]
Awesome work, thanks @rajbhargava. This is a big improvement. I have added a number of proposed changes, including the addition of probabilities as you suggest, to the [google doc](https://docs.google.com/document/d/1u-uK5HjYFCwNHzqz-bBc9AJgWXDb-pP8OzYzZU3ZqWI/edit#). 

I know you're busy until Friday, and I don't want to abuse your time — do you have any sense of whether it's feasible to make these changes today? (If not I'll try to do them myself by tonight, so that we can field and have some data for you to work with on Friday.) 

Thanks again!

**Comment by rajbhargava** [comment]
@bblockwood I won't be able to get to anything else before Friday.

**Comment by bblockwood** [comment]
Ok, thanks for the update.
